### PR TITLE
Use stdlib's plistlib

### PR DESCRIPTION
**Pull request by adrientetar**

### CODE REVIEW

File: Lib/ufoLib/glifLib.py [hunk 1]
```diff
@@ -739,7 +739,7 @@ def _writeAnchors(glyphObject, writer, identifiers):
 		writer.newline()
 
 def _writeLib(glyphObject, writer):
-	from ufoLib.plistlib import PlistWriter
+	from plistlib import PlistWriter
```
Comment (adrientetar) [inline Lib/ufoLib/glifLib.py]: There's no Plistwriter in the standard library..?

**Comment by moyogo** [comment]
For some reason PlistWriter was made private in Python 3 because they thought nobody was using it outside of plistlib.

**Comment by adrientetar** [comment]
They never intended it to be part of public API, it's always been absent from the doc.

**Comment by moyogo** [comment]
Good Point.
The API is too different, we need a py23 wrapper if we want to use the standard plistlib.

**Comment by adrientetar** [comment]
I'm on it.

**Comment by typesupply** [comment]
Yeah, Just wrote the original version of plistlib so he had an insider's knowledge of what was available inside of the module.

**Pull request merged**